src/Pages/Vote.js

```diff
@@ -1,12 +1,10 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import Footer from "../components/Footer/Footer";
+import Button from "../components/Button/Button";
 
 function Vote() {
-  return (
-    <button>
-      <Link to="/Result">Button</Link>
-    </button>
-  );
+  return <Footer>{<Button>{<Link to="/Result">Next</Link>}</Button>}</Footer>;
```

Comment:
omg this looks dirty... you could just use parentheses to have a multi-line `return`